packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol

```diff
@@ -11,6 +11,10 @@ import { OptimismMintableERC20 } from "../universal/OptimismMintableERC20.sol";
  * @title L2StandardBridge
  * @notice The L2StandardBridge is responsible for transfering ETH and ERC20 tokens between L1 and
  *         L2. ERC20 tokens sent to L1 are escrowed within this contract.
+ *         Note that this contract is not intended to support all variations of ERC20 tokens; this
```

Comment:
I'm a little confused by this note, is the "this includes" the set of tokens we do intend to support or the set of tokens we do not intend to support? I assume it's the latter. Maybe could be reworded like:

> Note that this contract is not intended to support all variations of ERC20 tokens. Examples of some token types that may not be properly supported by this contract include, but are not limited to: ...

Reply:
Your rewrite is good. Fixed.